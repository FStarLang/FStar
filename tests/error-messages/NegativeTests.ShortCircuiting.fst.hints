[ "Ìuz±?:×Q…\u001e\u0010 ‚øÝJ", [] ]