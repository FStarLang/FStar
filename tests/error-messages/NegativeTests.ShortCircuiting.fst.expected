>> Got issues: [
NegativeTests.ShortCircuiting.fst(21,16-21,33): (Error 19) Subtyping check failed; expected type b: Prims.bool{NegativeTests.ShortCircuiting.bad_p b}; got type Prims.bool; The SMT solver could not prove the query, try to spell your proof in more detail or increase fuel/ifuel (see also NegativeTests.ShortCircuiting.fst(19,31-19,38))
>>]
>> Got issues: [
NegativeTests.ShortCircuiting.fst(25,11-25,36): (Error 19) assertion failed (Also see: prims.fst(424,31-424,44))(Other related locations: prims.fst(421,71-421,83)); The SMT solver could not prove the query, try to spell your proof in more detail or increase fuel/ifuel
>>]
NegativeTests.ShortCircuiting.fst(19,4-19,7): (Warning 240) Admitting NegativeTests.ShortCircuiting.bad without a definition
NegativeTests.ShortCircuiting.fst(23,4-23,6): (Warning 240) Admitting NegativeTests.ShortCircuiting.ff without a definition
Verified module: NegativeTests.ShortCircuiting
All verification conditions discharged successfully
